Makefile

```diff
@@ -36,6 +36,9 @@ TIMEOUT ?= 3600s
 PACKAGES := go list ./... | grep -v /vendor | grep -v /out
 SOURCE_DIRS = cmd pkg test
 
+# defines non-default shell name to be used in godog steps which use instance of shell
+TEST_WITH_SHELL ?= default
```

Comment:
Is this value "default" needed? or does an empty string also work?

If so, it would have to be named `TEST_WITH_SPECIFIED_SHELL` or `OVERRIDE`, `OTHER`, `ALTERNATIVE`, etc... to indicate it is not using the default.

Reply:
@gbraad Both variants works, originally I used an empty string but then I have added default to make it easier to understand. Not sure whether this was a good move.

Please let me know which solution you would like to see and I will change it accordingly.

Reply:
The current way it is coded feels like this variable is a necessity. maybe it is the naming, maybe it is the value...

Reply:
@gbraad Ok, will use `TEST_WITH_SPECIFIED_SHELL` which will be without specifying anything just empty string.